launchers/snesmsu1

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+romname="${@%.zip}"
+cd .snesmsu1
+mkdir tmp
+fuse-zip -r "$@" tmp
+/usr/share/chimera/bin/launcher snesmsu1 snes9x "$HOME/.local/share/chimera/content/snesmsu1/.snesmsu1/tmp/$romname"
+fusermount -u tmp
```

Comment:
I am not sure these commands will be executed after exiting the game through the Steam overlay.

Reply:
So far it's working on my dev box. I was worried they might not too. Definitely would like to make sure it works on other installs though.